website/docs/api-layers.md

```diff
@@ -1347,17 +1337,17 @@ https://github.com/explosion/thinc/blob/master/thinc/layers/with_array.py
 <inline-list>
 
 - **Input:** <tt>Sequence[Sequence[Any]]</tt>
-- **Output:** <tt>List[Array2d]</tt>
+- **Output:** <tt>ListXd</tt>
 
 </inline-list>
 
 Flatten nested inputs on the way into a layer and reverse the transformation
 over the outputs.
 
-| Argument    | Type           | Description        |
-| ----------- | -------------- | ------------------ |
-| `layer`     | <tt>Model</tt> | The layer to wrap. |
-| **RETURNS** | <tt>Model</tt> | The wrapped layer. |
+| Argument    | Type                                                              | Description        |
+| ----------- | --------------................................................... | ------------------ |
```

Comment:
This seems like an editor problem, should be `-` instead of `.` (`prettier` can do this automatically btw)